From: Sven Roederer <devel-sven@geroedel.de>
Date: Sat, 3 Apr 2021 22:00:31 +0200
Subject: nut: fix typo in nutshutdown script

Even it's only cosmetic and should not affect the function of regular system,
fix the name of the IPKG_INSTROOT variable.
Typo was added long ago with 8400c9a6ec799.

Signed-off-by: Sven Roederer <devel-sven@geroedel.de>

diff --git a/net/nut/Makefile b/net/nut/Makefile
index c699fe260aec50b2f013351aa1edb0a690881159..c26c5832c24cab7126e1be912e7151a17ef4e8a8 100644
--- a/net/nut/Makefile
+++ b/net/nut/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=nut
 PKG_VERSION:=2.7.4
-PKG_RELEASE:=21
+PKG_RELEASE:=22
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://www.networkupstools.org/source/2.7/
diff --git a/net/nut/files/nutshutdown b/net/nut/files/nutshutdown
index 86fe01bb957e0eaad442830e21f5f16014f451f8..7270f983f1088e90b72dae4ca00e2e8dd9bc4370 100755
--- a/net/nut/files/nutshutdown
+++ b/net/nut/files/nutshutdown
@@ -24,7 +24,7 @@ do_fsd() {
 		mount -o remount,ro /overlay /overlay
 		mount -o remount,ro / /
 
-		. ${IPKG_INSTOOT}/lib/functions.sh
+		. ${IPKG_INSTROOT}/lib/functions.sh
 
 		if [ -f /etc/config/nut_server ]; then
 			config_load nut_server
